tests/libs/airgap.sh

```diff
@@ -30,6 +30,22 @@ function setup_airgap_registry_mirror() {
   local TO_CHANNEL=$4
 
   create_machine "$NAME" "$DISTRO" "$PROXY"
+
+  lxc exec "$NAME" -- bash -c "
+    mkdir -p /root/snap/microk8s/common
+    echo '
+---
+version: 0.1.0
+extraKubeletArgs:
```

Comment:
the dns addon should be setting these args, why do we set them explicitly here?

Reply:
skips an extra restart and lease timeouts for the kubernetes services, saves a few minutes per test